src/control/server/storage/bdev/backend_json.go

```diff
@@ -123,9 +124,7 @@ func defaultSpdkConfig() *SpdkConfig {
 		},
 		{
 			Method: SpdkBdevNvmeSetHotplug,
-			Params: NvmeSetHotplugParams{
-				PeriodUsec: HotplugPeriod,
-			},
+			Params: NvmeSetHotplugParams{},
```

Comment:
I guess the default config doesn't have hotplug enabled, so the period shouldn't be set, correct?

Reply:
right